token-lending/program/src/processor.rs

```diff
@@ -28,6 +28,7 @@ use solana_program::{
 use spl_token::state::{Mint, Account};
 use spl_token::solana_program::instruction::AccountMeta;
 use crate::state::FeeCalculation::Exclusive;
+use std::u64;
```

Comment:
This is not necessary, see https://github.com/solana-labs/solana-program-library/pull/1441#discussion_r612864626